src/components/Datepicker/sgds-datepicker.scss

```diff
@@ -0,0 +1,31 @@
+
+:host {
+  --datepicker-theme-color: var(--sgds-primary);
+  --datepicker-hover-bg-color:  var(--sgds-primary-100);
```

Comment:
i thought we will be using sass function to make it 25% darker/lighter than --sgds-primary ?  for var(--sgds-primary-100) and sgds-primary-600